store/postgres/replay_repository.go

```diff
@@ -265,3 +265,29 @@ func (repo *replayRepository) GetByProjectIDAndStatus(projectID uuid.UUID, statu
 	}
 	return replaySpecs, nil
 }
+
+func (repo *replayRepository) GetByProjectID(projectID uuid.UUID) ([]models.ReplaySpec, error) {
+	var replays []Replay
+	if err := repo.DB.Preload("Job").Joins("JOIN job ON replay.job_id = job.id").
+		Where("job.project_id = ? and replay.created_at >= (NOW() - INTERVAL '1 MONTH')", projectID).Order("created_at DESC").Find(&replays).Error; err != nil {
```

Comment:
We can either take this duration as argument to this function or just return all, hardcoding this doesn't look good

Reply:
agree. since the function name is GetByProjectID, I will just return them all and filter for the recent ones in the caller